tests/Stuart/tests/ClientTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Stuart\Tests;
+
+use GuzzleHttp\Psr7\Response;
+use Stuart\Client;
+use Stuart\Infrastructure\Authenticator;
+
+class ClientTest extends \PHPUnit_Framework_TestCase
+{
+    private $authenticator;
+    private $httpClient;
+
+    public function setUp()
+    {
+        $this->httpClient = \Phake::mock(\GuzzleHttp\Client::class);
+        $this->authenticator = \Phake::mock(Authenticator::class);
+
+        \Phake::when($this->httpClient)->request(\Phake::anyParameters())->thenReturn(
+            new Response(200, [], null)
+        );
+    }
+
```

Comment:
I would add another test case in order to verify the `performPost` method is called with the correct parameters, using:

```php
public function test_http_call_with_correct_parameters() {
  ...
  \Phake::verify($this->httpClient)->performPost('', '/v2/jobs/123/cancel');
  ...
}
```